### PR TITLE
Final app service API typing

### PR DESCRIPTION
**Pull request by mike12345567**
## Description
As title, the finishes off the API typing for the app service, in line with the two PRs before it:

1. https://github.com/Budibase/budibase/pull/15100
2. https://github.com/Budibase/budibase/pull/15112

This contains APIs like views, webhooks and tables.

### CODE REVIEW

File: packages/backend-core/src/middleware/auditLog.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { BBContext } from "@budibase/types"
+import { Ctx } from "@budibase/types"
 
-export default async (ctx: BBContext | any, next: any) => {
+export default async (ctx: Ctx, next: any) => {
```
Comment (samwho) [inline packages/backend-core/src/middleware/auditLog.ts:3]: Can we not just delete this? It does nothing.

File: packages/server/src/api/routes/script.ts [hunk 2]
```diff
@@ -1,10 +0,0 @@
-import Router from "@koa/router"
```
Comment (samwho) [inline packages/server/src/api/routes/script.ts:1]: I'm assuming this was unused?

**Review by samwho — APPROVED** [review]

File: packages/backend-core/src/middleware/auditLog.ts (hunk 1 above)
Comment (mike12345567) [reply packages/backend-core/src/middleware/auditLog.ts:3]: It is a place holder that is overriden by pro - its one of the oddities that we have to deal with due to the closed source submodule.

File: packages/server/src/api/routes/script.ts (hunk 2 above)
Comment (mike12345567) [reply packages/server/src/api/routes/script.ts:1]: Yeap - theres no API that uses this and it didn't even really function properly/was even tested - it didn't serve a purpose and probably never would!

**Pull request merged**